mods/ts/rules/aircraft.yaml

```diff
@@ -78,12 +78,13 @@ ORCA:
 		RearmBuildings: gahpad, nahpad
 		ROT: 5
 		Speed: 186
+		MoveIntoShroud: false
 	Health:
 		HP: 200
 	Armor:
 		Type: Light
 	RevealsShroud:
-		Range: 10c0
+		Range: 2c0
```

Comment:
I expect that this will make it possible to creep them into the shroud.  If we're going to have `MoveIntoShroud: false` to match the original game, then we should remove the sight range to also match the original game.